### ISSUE TITLE
How to prevent some pages from translation? (e.g. all blog articles)

### ISSUE THREAD

**Issue by herrvigg** [herrvigg/qtranslate-x#549]
<a href="https://github.com/KarpIvan"><img src="https://avatars2.githubusercontent.com/u/32894338?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [KarpIvan](https://github.com/KarpIvan)**
_Wednesday Oct 18, 2017 at 07:45 GMT_
_Originally opened as https://github.com/qTranslate-Team/qtranslate-x/issues/549_

----

Hi, I have some problems in using the qTranslate plugin. It would be great if someone will be able to help!

1) how to prevent pages from translation. For example, I have www.mysite.com/blog/article1, and I don't wish to translate this article and have the URL like www.mysite.com/blog/de/article1 (because it has the same content (not translated) duplicate))

in short: all posts from my blog I don't want to translate.

2) how to use this plugin with wp seo yoast? now I can't

Thanks a lot in advance!
